src/main/webapp/resources/js/components/Tables/filter-utilities.js

```diff
@@ -0,0 +1,29 @@
+import uniqBy from "lodash/uniqBy";
```

Comment:
File comment

Reply:
Updated in d5afbb8fb56a28350db36d77116c7a41f5c993dd